beancount/query/query_compile.py

```diff
@@ -204,13 +215,13 @@ def __init__(self, left, right):
 class EvalAdd(EvalBinaryOp):
 
     def __init__(self, left, right):
-        f = lambda x, y: Decimal(x + y)
+        f = lambda x, y: (x + y)
```

Comment:
Redundant parentheses.

Reply:
Personally, I find this easier to read, but happy to change if you like.